src/Symfony/Component/ErrorHandler/Debug.php

```diff
@@ -58,3 +58,8 @@ public static function enable($errorReportingLevel = E_ALL, $displayErrors = tru
         DebugClassLoader::enable();
     }
 }
+
+// to be removed in 5.0
+if (!class_exists(\Symfony\Component\Debug\Debug::class)) {
```

Comment:
I think you can add `false` as the 2nd arg because if the class has not been loaded yet, it's perfectly fine to use it as the alias target.

Reply:
That'd mean the `if` condition is useless, isn't it? This file will *always* be autoloaded first (by composer `autoload-dev > files`).

Reply:
We are adding this condition for that one person that uses the standalone components without composer ^^ Or the one that has a previous autoload file that already load the `Debug\Debug` class.

Also shouldn't it be in `autoload` ? `autoload-dev` is root only.

Reply:
This logic should be used for development only, that's why I moved it to `autoload-dev`.

Reply:
Nope: dev != debug

Reply:
fair enough, fixed.